src/components/image/index.js

```diff
@@ -88,6 +96,25 @@ class Image extends PureBaseComponent {
       />
     );
   }
+
+  onLayout = (event) => {
```

Comment:
Why do we need that?
The overlay will be stretched anyway because you're using absoluteFillObject in the overlay component.

Reply:
Since it's not stretching as you expect

Reply:
It looks the same with or without the width